content/about/open-letter-meta-dma/technical-briefing.md

```diff
@@ -0,0 +1,271 @@
+- In contrast, **Threads is struggling against Twitter/X and Bluesky**—so Meta is embracing federation as a way to level the playing field and gain market share.  
+
+In other words, **Meta embraces open protocols when it benefits from them, but resists them when they threaten its market control**.  
+
+If Meta were genuinely committed to **interoperability**, it would recognize **XMPP as the most viable and proven protocol for federated messaging**—just as it has in the past.
+
+## Building the Future of Messaging
+
+Federation **is not a security risk—it is a scalable, privacy-friendly model** that has already been **proven in global communication networks** like **email and federated social media**. By leveraging the **XMPP server-to-server (s2s) protocol**, messaging services can **achieve true interoperability** while maintaining **security, user privacy, and control**.
+
+### A Call to Meta: Build Your Interoperability Stack on XMPP
+
+We at the XMPP Standards Foundation (XSF) urge Meta to build its interoperability framework on top of XMPP federation.  
+
+If Threads can implement the Fediverse protocol, there is no reason why Meta cannot do the same with XMPP for Facebook Messenger and WhatsApp—especially since WhatsApp itself was originally built on XMPP.
+
+### The XSF Welcomes Collaboration
+
+The XSF is an open and inclusive organization, welcoming contributions from companies, developers, vendors, and network operators. We recognize that, while Messenger and WhatsApp were originally based on XMPP, there may still be technical discussions needed to align the protocol with Meta’s current needs.
+
+If Meta has specific requirements, such as:
+
+- Features unique to Facebook Messenger or WhatsApp
+- Scalability considerations due to the size of Meta’s user base  
+- New extensions to address Meta’s specific use cases  
+
+The XSF is the right place to have these discussions. Our mission is to evolve and improve federated messaging standards.
+
+### Join the Conversation on the Future of Federated Messaging
+
+We understand that Meta may wish to contribute to the development of XMPP, and we would welcome Meta’s most prominent messaging experts to bring their expertise, concerns, and ideas into the work of the XMPP Standards Foundation.
+
+Let’s work together to build a truly interoperable, open, and competitive messaging future.
+
+***
+
+### Contact Information
+
+Press and general inquiries: [info@xmpp.org](mailto:info@xmpp.org)
+
+Ralph Meijer  
+[Chair of the Board](/about/xsf/people#chair)  
+XMPP: [ralphm@ik.nu](xmpp:ralphm@ik.nu)  
+Mail: [ralphm@ik.nu](mailto:ralphm@ik.nu)
+
+Matthew Wild  
+[Executive Director](/about/xsf/people#executive-director)  
+XMPP: [me@matthewwild.co.uk](xmpp:me@matthewwild.co.uk)  
+Mail: [me@matthewwild.co.uk](mailto:me@matthewwild.co.uk)
+
+Mickaël Rémond  
+Member and Author  
```

Comment:
We should give Mickael a special title. Liaison or something.

Reply:
In be6b0a53.